### PR TITLE
docs: changelog for 4.20.7

### PR DESCRIPTION
**Pull request by PeachScript**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

- 🐞 Fix form instance lost bug when open Drawer. [#35706](https://github.com/ant-design/ant-design/pull/35706) [@crazyair](https://github.com/crazyair)
- 🐞 Fix the invalid space between icon and text when using the icon prop in Segmented's options. [#35701](https://github.com/ant-design/ant-design/pull/35701)
- 💄 Optimize arrow style of Popover. [#35717](https://github.com/ant-design/ant-design/pull/35717)
- TypeScript
  - 🤖 Fix type hints problem for Card. [#35753](https://github.com/ant-design/ant-design/pull/35753)

---

- 🐞 修复 Drawer 打开时 form 实例为 null 的问题。[#35706](https://github.com/ant-design/ant-design/pull/35706) [@crazyair](https://github.com/crazyair)
- 🐞 修复 Segmented 组件中选项使用 icon 属性时图标与文字之间的间距失效问题。[#35701](https://github.com/ant-design/ant-design/pull/35701)
- 💄 优化 Popover 的箭头效果。[#35717](https://github.com/ant-design/ant-design/pull/35717)
- TypeScript
  - 🤖 修复 Card 组件的类型提示。[#35753](https://github.com/ant-design/ant-design/pull/35753)

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -15,6 +15,17 @@ timeline: true
...
 ---
 
+## 4.20.7
+
+`2022-05-30`
+
+- 🐞 Fix form instance lost bug when open Drawer. [#35706](https://github.com/ant-design/ant-design/pull/35706) [@crazyair](https://github.com/crazyair)
+- 🐞 Fix the invalid space between icon and text when using the icon prop in Segmented's options. [#35701](https://github.com/ant-design/ant-design/pull/35701)
+- 💄 Optimize arrow style of Popover. [#35717](https://github.com/ant-design/ant-design/pull/35717)
+
```
Comment (xrkffgg) [inline CHANGELOG.en-US.md:25]:
```suggestion
```

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -15,6 +15,17 @@ timeline: true
...
 ---
 
+## 4.20.7
+
+`2022-05-30`
+
+- 🐞 修复 Drawer 打开时 form 实例为 null 的问题。[#35706](https://github.com/ant-design/ant-design/pull/35706) [@crazyair](https://github.com/crazyair)
+- 🐞 修复 Segmented 组件中选项使用 icon 属性时图标与文字之间的间距失效问题。[#35701](https://github.com/ant-design/ant-design/pull/35701)
+- 💄 优化 Popover 的箭头效果。[#35717](https://github.com/ant-design/ant-design/pull/35717)
+
```
Comment (xrkffgg) [inline CHANGELOG.zh-CN.md:25]:
```suggestion
```

File: CHANGELOG.en-US.md [hunk 3]
```diff
@@ -15,6 +15,17 @@ timeline: true
 
 ---
 
+## 4.20.7
+
+`2022-05-30`
+
+- 🐞 Fix form instance lost bug when open Drawer. [#35706](https://github.com/ant-design/ant-design/pull/35706) [@crazyair](https://github.com/crazyair)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md:22]: 下面几个都把 Component 名字提到文字最前面~

Comment (PeachScript) [reply CHANGELOG.en-US.md]: 已更新~

**Pull request merged**